content/millennium/dstu2/devices/device.md

```diff
@@ -7,10 +7,31 @@ title: Device | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Device Resource implementation is currently limited to devices implanted in a patient such as a pacemaker or insulin pump. As such, this resource currently exposes the known implant history for a patient at this site.
+
+The following fields are returned if valued:
+
+* [Device id](http://hl7.org/fhir/dstu2/resource-definitions.html#Resource.id){:target="_blank"}
+* [Type](http://hl7.org/fhir/DSTU2/device-definitions.html#Device.type){:target="_blank"}
+* [Manufacturer](http://hl7.org/fhir/DSTU2/device-definitions.html#Device.manufacturer){:target="_blank"}
+* [Model](http://hl7.org/fhir/DSTU2/device-definitions.html#Device.model){:target="_blank"}
+* [Manufacture date](http://hl7.org/fhir/DSTU2/device-definitions.html#Device.manufactureDate){:target="_blank"}
+* [Expiration date](http://hl7.org/fhir/DSTU2/device-definitions.html#Device.expiry){:target="_blank"}
+* [udi (FDA Unique Device Identifier)](http://hl7.org/fhir/DSTU2/device-definitions.html#Device.udi){:target="_blank"}
```

Comment:
Nit?: On the actual FHIR documentation and on the [FDA site](https://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/UniqueDeviceIdentification/default.htm), the abbreviation is all uppercase. Since we are returning human friendly names, should we do the same here?

Also, it looks weird :)